### PR TITLE
organize and add SKT carrier packages

### PR DESCRIPTION
**Pull request by ghost**
sorted the list

### CODE REVIEW

File: resources/assets/uad_lists.json [hunk 1]
```diff
@@ -13495,6 +13495,15 @@
     "labels": [],
     "removal": "Advanced"
   },
+  {
+    "id": "com.samsung.android.app.dtv.dmb",
```
Comment (ghost) [inline resources/assets/uad_lists.json]: dmb isn't related to SKT btw

**Comment by AnonymousWP** [comment]
Hi, to prevent you from doing double (unnecessary) work, you could check the current PRs as they contain a lot of changes.

**Comment by ghost** [comment]
I will do that tomorrow since it's 12am here

**Comment by AnonymousWP** [comment]
Why did you close it?

**Pull request closed without merging**